Node/package.json

```diff
@@ -23,6 +23,6 @@
     "mocha": "^5.2.0",
     "nodemon": "^1.18.9",
     "restify": "^4.3.1",
-    "typescript": "^2.5.3"
+    "typescript": "^3.3.3333"
```

Comment:
Is this version number right?

Reply:
I'm having TS1039 issues resulting in compile error. Updating to this version seems resolving it. Check out ref: https://github.com/awslabs/aws-cdk/issues/1075

Reply:
That's just a weird version number but it checks out!